src/connect.rs

```diff
@@ -72,15 +72,17 @@ pub fn connect_to(peer_info: CrustInfo, send_after_connect: Option<WireMsg>) ->
 
 fn handle_new_connection_res(
     peer_addr: SocketAddr,
-    new_peer_conn_res: Result<quinn::NewClientConnection, quinn::ConnectionError>,
+    new_peer_conn_res: Result<quinn::NewConnection, quinn::ConnectionError>,
 ) {
     let new_peer_conn = match new_peer_conn_res {
         Ok(new_peer_conn) => new_peer_conn,
         Err(e) => return handle_connect_err(peer_addr, &From::from(e)),
     };
+    current_thread::spawn(new_peer_conn.driver.map_err(|_| ()));
```

Comment:
In case of error I would like to destroy the connection object to prevent leaks and limbo states. I don't know what this error means but i guess dropping it is the safest way to go ?